pkg/kubelet/network/dns/dns_test.go

```diff
@@ -52,6 +52,40 @@ var (
 	}
 )
 
+func newFakeConfigurer(recorder record.EventRecorder, nodeRef *v1.ObjectReference, nodeIPs []net.IP, clusterDNS []net.IP, clusterDomain, resolverConfig string) *Configurer {
+	return &Configurer{
+		recorder:         recorder,
+		getHostDNSConfig: getResolvConfHostDNSConfig,
```

Comment:
This looks like it only tests the implementation in this file?  Should this be `getHostDNSConfig` based on the GOOS selected?

Reply:
Done